cmd/server-main.go

```diff
@@ -144,18 +144,20 @@ func initConfig() {
 
 // Generic Minio initialization to create/load config, prepare loggers, etc..
 func minioInit(ctx *cli.Context) {
-	// Is TLS configured?.
-	globalIsSSL = isSSL()
-
-	// Initialize minio server config.
-	initConfig()
-
 	// Enable all loggers by now so we can use errorIf() and fatalIf()
 	enableLoggers()
```

Comment:
How would one enable logging before initializing a config file? and what happens after the config has been initialized? - initConfig() also loads previously saved config - you would never set the logging properly with this approach.